whenchanged/whenchanged.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, files, command, recursive=False, run_once=False,
         self.last_run = 0
         self.verbose_mode = verbose_mode
         self.process_env = os.environ.copy()
+        self.last_event_type = ''
```

Comment:
Now that we have last_event_type, can we get rid of the regex on line 95?

Reply:
Could become risky in case the last event type and the current event env var evolve separately.
But at this time this should work.
What do you prefer?